src/components/TitlesCarousel/TitlesCarousel.js

```diff
@@ -1,7 +1,8 @@
 import { obseverDom } from '../../shared/observer';
 import { MovieTile } from '../Tiles/MovieTile';
+import { Button } from '../Button/Button';
```

Comment:
I don't see any check if we actually have any data?

Generally you should add something like
`if(!movies.length) {return ''}`
Otherwise we would get a console error (yeah, cuz js wouldn't check anything in runtime 8) )

If we don't have any movies, like API wouldn't return anything, or we get an error or anything, we should return just an empty string in this component

Reply:
All comments included.